src/utils/log.js

```diff
@@ -45,7 +47,7 @@ const banner = `
 | | | / __| | '_ \\ / _\` |   | |/ _ \\ '_ \` _ \\| '_ \\| |/ _\` | __/ _ \\
 | |_| \\__ \\ | | | | (_| |   | |  __/ | | | | | |_) | | (_| | ||  __/
  \\___/|___/_|_| |_|\\__, |   |_|\\___|_| |_| |_| .__/|_|\\__,_|\\__\\___|
-                   |___/                     |_|                    
+                   |___/                     |_|              v${pack.version}
```

Comment:
we should also look at including the name and version of the template

Reply:
we are leaving this for now